ui.tests/test-module/specs/wizard/wizard.runtime.spec.js

```diff
@@ -39,12 +39,14 @@ describe("Form with Wizard Layout Container", () => {
         cy.get(".cmp-adaptiveform-wizard__wizardpanel").eq(1).should('not.have.class', 'cmp-adaptiveform-wizard__wizardpanel--active');
     });
 
-    it("verify Next Navigation Button Functionality", () => {
+    it("verify Next and Previous Navigation Button Functionality", () => {
```

Comment:
please add a separate collateral and test the visibility in a separate test case.